### PR TITLE
Add logger to the project

### PR DESCRIPTION
**Pull request by AdriOli134**
Add the At18Logger class in the new folder common and implement it as an example in the ProcessFrameController and the ObjectDetectionModel
Add the loggin properties to the application.properties file.
The log file (java0.log) creates in src folder and it is updated every time the program finishes executing

### CODE REVIEW

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectDetection/ObjectDetectionModel.java [hunk 1]
```diff
@@ -19,8 +19,9 @@
 import ai.djl.training.util.ProgressBar;
 import ai.djl.translate.TranslateException;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.jalau.at18.searchobject.common.logger.At18Logger;
+//import org.slf4j.Logger;
```
Comment (junior0123) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectDetection/ObjectDetectionModel.java]: Perhaps you could remove the commented lines of code if they are not necessary.

File: searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java [hunk 2]
```diff
@@ -36,7 +39,7 @@ public ResponseEntity<List<MatchInfo>> readDataCriteriaFrame(@RequestParam("file
                                                                  @RequestParam String recipient) throws IOException {
         // save file, it is saving the zip file and getting the path
         Path path = storageService.save(file);
-        System.out.println(path);
+        LOG.info(path.toString());
         // get the route file
         UnzipFile unzip = new UnzipFile(path);
         //FileSource fileSource = new FileSource(path.toFile().getAbsolutePath());
```
Comment (junior0123) [inline searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java]: Perhaps you could remove the commented lines of code if they are not necessary.

**Review by RodrigoValda — APPROVED** [review]
Good Job

**Review by junior0123 — CHANGES_REQUESTED** [review]
Great integration work, you could check the small corrections.

File: searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java (hunk 2 above)
Comment (AdriOli134) [reply searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java]: I didn't do this comments, but i can still remove them if you prefer.

Comment (junior0123) [reply searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java]: If you could Please :D

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectDetection/ObjectDetectionModel.java (hunk 1 above)
Comment (AdriOli134) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectDetection/ObjectDetectionModel.java]: It is already removed

File: searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java (hunk 2 above)
Comment (AdriOli134) [reply searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java]: It is already removed

**Review by junior0123 — APPROVED** [review]

**Pull request merged**